submissions/krylenger/dom-api-task/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Motorcycles</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <input id='nav__toggle' type="checkbox" class="nav__toggle">
```

Comment:
It's better to use `button` tag here instead of `input` and `label`. It will be better for semantic rules.

Reply:
✅ done 
please double check the way how I've implemented this